src/main/java/org/broadinstitute/hellbender/tools/exome/alleliccount/AllelicCountReader.java

```diff
@@ -69,9 +69,9 @@ static AllelicCount createAllelicCountWithVerbosity(final DataLine dataLine, fin
         if (verbosity == AllelicCountTableColumn.AllelicCountTableVerbosity.BASIC) {
             return new AllelicCount(interval, refReadCount, altReadCount);
         } else {
-            final Nucleotide refNucleotide = Nucleotide.valueOf(dataLine.get(AllelicCountTableColumn.REF_NUCLEOTIDE.name()).getBytes()[0]);
-            final Nucleotide altNucleotide = Nucleotide.valueOf(dataLine.get(AllelicCountTableColumn.ALT_NUCLEOTIDE.name()).getBytes()[0]);
-            final int readDepth = dataLine.getInt(AllelicCountTableColumn.READ_DEPTH.name());
+            final Nucleotide refNucleotide = Nucleotide.valueOf(dataLine.get(AllelicCountTableColumn.REF_NUCLEOTIDE).getBytes()[0]);
```

Comment:
I think we should keep `name()` here just to be safe (i.e., in case someone goes and adds different names to `AllelicCountTableColumn` in the future).